src/Workspaces/CoreTest/SolutionTests/SolutionWithSourceGeneratorTests.cs

```diff
@@ -372,7 +371,7 @@ public async Task CompilationsInCompilationReferencesIncludeGeneratedSourceFiles
         [Fact]
         public async Task RequestingGeneratedDocumentsTwiceGivesSameInstance()
         {
-            using var workspace = CreateWorkspaceWithRecoverableSyntaxTreesAndWeakCompilations();
+            using var workspace = CreateWorkspace();
```

Comment:
This test can be deleted now; admittedly we probably should have deleted this when the compilation went non-lazy, since if you look at the comment before the AssertHeld it wasn't even updated properly.

Reply:
deleted